### ISSUE TITLE
The fullscreen option isn't working on iOS

### ISSUE THREAD

**Issue by adambutler** [TheWidlarzGroup/react-native-video#2962]
# Bug

## Platform

- [x] iOS - Tested broken on iOS 16.1
- [ ] Android - Tested - Not expecting to work as `fullscreen` prop is iOS only
- [ ] Windows UWP - Untested - Not expecting to work as `fullscreen` prop is iOS only
- [ ] Windows WPF - Untested - Not expecting to work as `fullscreen` prop is iOS only

## Environment info

React native info output:

```bash
System:
    OS: macOS 13.0.1
    CPU: (10) arm64 Apple M1 Max
    Memory: 544.30 MB / 64.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 14.19.0 - ~/.asdf/installs/nodejs/14.19.0/bin/node
    Yarn: 1.22.19 - ~/.asdf/shims/yarn
    npm: 6.14.16 - ~/.asdf/plugins/nodejs/shims/npm
    Watchman: 2022.11.14.00 - /opt/homebrew/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /Users/adambutler/.asdf/shims/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 22.2, iOS 16.2, macOS 13.1, tvOS 16.1, watchOS 9.1
    Android SDK: Not Found
  IDEs:
    Android Studio: 2021.2 AI-212.5712.43.2112.8609683
    Xcode: 14.2/14C18 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.15 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.1.0 => 18.1.0 
    react-native: 0.70.6 => 0.70.6 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

Library version: `6.0.0-alpha.4`

## Steps To Reproduce

1. Add a `fullscreen={true}` prop onto a `<Video>` component
...

## Expected behaviour

1. I expect the video player to enter into fullscreen on play on iOS but it does not.

## Reproducible sample code

See repo: https://github.com/adambutler/bug-demo-react-native-video-fullscreen

```js
import React from 'react';
import Video from 'react-native-video';

const App = () => {
  return (
    <Video
      source={{uri: 'https://vjs.zencdn.net/v/oceans.mp4'}}
      fullscreen={true}
      style={{
        position: 'absolute',
        top: 0,
        left: 0,
        bottom: 0,
        right: 0,
      }}
    />
  );
};

export default App;
```

## Video sample

Any video but I've been using `https://vjs.zencdn.net/v/oceans.mp4` for testing.

**Comment by johnbillion** [comment]
This is likely the same issue as #2808, see the most recent comments for a patch.

**Comment by freeboub** [comment]
This issue should be fixed on last V6.0.0-alpha.8 version.
please let me know if you still have the issue
